configs/rbac_cel_custom_library.yaml

```diff
@@ -0,0 +1,196 @@
+# RBAC with policies based on CEL expressions with custom library
```

Comment:
I think maybe these should live in `examples/` rather than here, that way we can test them and provide a package experience. Not sure, @phlax any opinions?